src/components/KpiLink.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <router-link
-    v-tooltip="kpi.name"
+    v-tooltip="kpi.description? kpi.description : kpi.name"
```

Comment:
Liten sak, men vil ikke eslint klage på en manglende space på hver side av `?`? Vi har for så vidt litt å gjøre på den fronten uansett. 😄  `error  Operator '?' must be spaced  space-infix-ops`